### DISCUSSION TITLE
Is there an easy way to change the position of Navbar.brand Logo

### DISCUSSION THREAD

**Discussion by richardsondx** [chakra-ui/chakra-ui#5396] [Q&A]
Do you have an easy way to move the Logo to the center of the [Navbar](https://pro.chakra-ui.com/components/marketing/navbars/navbar-with-buttons.html?color=blue)? 

e.g:
`<Navbar.Brand position="center">`

When I move the code block after the Navlink items; it disappears.

**Comment by primos63** [comment]
You may want to send your question to the support email, support@chakra-ui.com.
This repo is for the opensource community version.
